src/js/components/CheckBoxGroup/index.d.ts

```diff
@@ -15,6 +15,7 @@ export interface CheckBoxType
 
 export interface CheckBoxGroupProps {
   value?: (number | string)[];
+  defaultValue?: (number | string | object)[];
```

Comment:
Why does `defaultValue` not match `value`?

Reply:
Good catch, fixed.

Reply:
not seeing the change yet, here or in propTypes.js